data/analysis.json

```diff
@@ -28,5 +28,11 @@
     "file_name": "AdministracionEfectivo.ear",
     "target": "eap8",
     "source": ""
+  },
+  "fuse-kafka":{
```

Comment:
The application is called `karaf`, not `kafka` (`FUSE-7.12.1/fuse-karaf-quickstarts-7.12.1.fuse-7_12_1-00009-redhat-00001.zip`) I got it wrong when I created the task. Could you please rename it?

And a nitpick: there is a missing whitespace between `:` and `{`

Reply:
Done